### PR TITLE
Integrated eq-smoke-tests into pipeline

### PR DESCRIPTION
**Pull request by jonnyshaw89**
Integrated eq-smoke-tests into pipeline
Added failure notifications via slack

## To Review
See that the smoke tests are being run as part of the pipeline.
http://concourse.dev.eq.ons.digital/teams/main/pipelines/eq

### CODE REVIEW

File: build_containers/author-build.dockerfile [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 
 RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
     && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list \
-    && curl -sL https://deb.nodesource.com/setup_7.x | bash -
+    && curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+    && sh -c 'echo "deb https://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
 
 RUN apt-get update \
-    && apt-get install -y nodejs yarn
+    && apt-get install -y --allow-unauthenticated nodejs yarn google-chrome-stable
```
Comment (ajmaddaford) [inline build_containers/author-build.dockerfile]: Any reason not to add the [Google GPG key](https://www.google.com/linuxrepositories/) so we don't need to use `--allow-unauthenticated `?

File: eq.yml [hunk 2]
```diff
@@ -573,38 +615,80 @@ jobs:
...
+        channel: '#eq-runner #eq-author'
+        attachments:
+          - pretext: Staging smoke-tests Failed
+            color: '#ff0000'
+            title: Concourse Build $BUILD_ID
+            title_link: http://concourse.dev.eq.ons.digital/builds/$BUILD_ID
 
 - name: staging-destroy
-  serial_groups: [staging-deploy, staging-functional-tests, staging-accessibility-tests, staging-performance-tests, staging-security-tests, staging-destroy]
+  serial_groups: [staging-deploy, staging-accessibility-tests, staging-performance-tests, staging-security-tests, staging-destroy]
```
Comment (ajmaddaford) [inline eq.yml]: We should remove references to accessibility, performance and security tests if they don't exist yet

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

**Review by MIslam92 — CHANGES_REQUESTED** [review]
Would there be any value changing the name of eq.yml to pipeline/eq-pipeline.yml as thats what is essentially is?

**Comment by jonnyshaw89** [comment]
@MIslam92 potentially but its very trivial

**Review by ajmaddaford — APPROVED** [review]

**Review by MIslam92 — APPROVED** [review]
"Squishy Squashy"

**Pull request merged**